EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -1206,297 +1206,297 @@ QUEST_LV_0300_20160310_001205	What do we do with the useless scriptures?{nl}We u
-QUEST_LV_0300_20160310_001316	How could we lose the monastery to the heresy like the religious body of the truthful tree..{nl}I feel sorry to the goddess and thank her.
-QUEST_LV_0300_20160310_001317	I won't be deceived by the lies of the goddess anymore.{nl}You also think so right?
-QUEST_LV_0300_20160310_001318	You are really.. {nl}You might be the envoy that was sent by the goddess..{nl}
-QUEST_LV_0300_20160310_001319	Strangely, Dominicas went to Pagarma chapel so often.{nl}He may have hid something important there.{nl}
-QUEST_LV_0300_20160310_001320	If you hide there, you will definitely get Dominicas.{nl}I will pray for you in the name of the goddess.
-QUEST_LV_0300_20160310_001321	As we cleaned up the monastery..{nl}I have full of the soul marbles that were left by Dominicas.{nl} 
-QUEST_LV_0300_20160310_001322	Can you remember when you broke the evil idol?{nl}How about I fill in the souls of the demons into the marbles and purify them.{nl}
-QUEST_LV_0300_20160310_001323	I will give you the soul marbles so please defeat the demons with these.{nl}You can just fill in the souls into the marbles.
-QUEST_LV_0300_20160310_001324	I will become Alchemist Master one day!
-QUEST_LV_0300_20160310_001325	Until I become an Alchemist Master, I am going to receive help of Bidotas.{nl}This is not because I lost to Bidotas, but I am just trying to use him!{nl}
+QUEST_LV_0300_20160310_001302	Good. You should do it.{nl}I was actually going to Veyrhas Hall to burn the scripture...{nl}
+QUEST_LV_0300_20160310_001303	It is better to see it once than to hear it a hundred times.{nl}Let's go together. Brother.
+QUEST_LV_0300_20160310_001304	I didn't know that one of people of the Winterspoon family came to Nahash Forest.{nl}Even so, since you know this person, he or she must be reliable.{nl}I am counting on you.
+QUEST_LV_0300_20160310_001305	I can't exactly remember what I did back then.{nl}As if I was possessed by something...{nl}
+QUEST_LV_0300_20160310_001306	Yes! {nl}I wouldn't have done such an impious thing with a conscious mind.
+QUEST_LV_0300_20160310_001307	I will apologize to the other monks later.{nl}I can't believe what I did.{nl}
+QUEST_LV_0300_20160310_001308	Look around you.{nl}Don't you think it's beautiful?{nl}
+QUEST_LV_0300_20160310_001309	The red blood of the demons contains special powers.{nl}That's why only their blood can wash away the corrupted power of the goddess.
+QUEST_LV_0300_20160310_001310	The Tree of Truth sect turned the monastery into a complete mess...
+QUEST_LV_0300_20160310_001311	We have to clean up the blood of the demons and tidy up the scripture...{nl}We have many things to do.
+QUEST_LV_0300_20160310_001312	It will be effective!{nl}Even so... If it goes wrong, you should tell me... For sure...
+QUEST_LV_0300_20160310_001313	Revelator, I am gratefully thankful to you.
```

Comment:
"gratefully thankful" is redundant.  Maybe "eternally thankful" or "Revelator, I owe you my eternal thanks." could work.